py/trtorch/csrc/tensorrt_backend.cpp

```diff
@@ -27,7 +27,7 @@ c10::impl::GenericDict TensorRTBackend::compile(c10::IValue mod_val, c10::impl::
   mod = core::lowering::LowerModule(mod);
 
   auto spec = c10::impl::toTypedDict<std::string, at::IValue>(method_compile_spec);
-  lowering::LowerInfo lower_info;
+  core::lowering::LowerInfo lower_info;
```

Comment:
pushed this commit during our call. Rebase required maybe ? will not conflict I guess anyways